sdks/web-sdk/src/lib/pages/DocumentStart.svelte

```diff
@@ -32,8 +34,7 @@
       await navigator.mediaDevices.getUserMedia({ video: true });
```

Comment:
@sFiFty consider extracting the camera logic to a single place, something like "camera-manager"